jwql/website/apps/jwql/static/js/jwql.js

```diff
@@ -520,10 +520,19 @@ function update_thumbnails_page(inst, proposal, base_url) {
         url: base_url + '/ajax/' + inst + '/archive/' + proposal + '/',
         success: function(data){
             // Perform various updates to divs
+            let t1 = performance.now();
```

Comment:
Are we logging this information now or was this for testing the loading speeds?

Reply:
Leftover from testing. Removed. I am excited that I now know how to print information to the console from javascript functions though.